rollup.config.js

```diff
@@ -13,80 +12,31 @@ function removeDefs() {
   };
 }
 
-const babelPreset = [['@babel/preset-env', { targets: { ie: '10' } }]];
-
-function getBabelPlugin() {
-  return getBabelOutputPlugin({
-    presets: babelPreset,
-    allowAllFormats: true,
-  });
-}
-
 export default async function () {
   await del('dist');
 
-  return [
-    // Main builds
-    {
-      input: 'src/index.ts',
-      plugins: [simpleTS('./')],
-      output: [
-        {
-          file: 'dist/esm/index.js',
-          format: 'es',
-        },
-        {
-          file: 'dist/cjs/index.cjs',
-          format: 'cjs',
-          exports: 'default',
-        },
-        {
-          file: 'dist/iife/index-min.js',
-          format: 'iife',
-          name: 'idbReady',
-          esModule: false,
-          plugins: [
-            terser({
-              compress: { ecma: 2020 },
-            }),
-            removeDefs(),
-          ],
-        },
-      ],
-    },
-    // Compat builds
-    {
-      input: 'src/index.ts',
-      external: (id) => {
-        if (id.startsWith('@babel/runtime')) return true;
+  return {
+    input: 'src/index.ts',
+    plugins: [simpleTS('./'), terser()],
```

Comment:
Would you recommend minifying here? I tend to only minify the iife version, since it's the one likely to be used without a bundler.

It's a bit of a pain during dev if you end up with something throwing in minified code.

Reply:
TBH there are good logical arguments in both directions. My main reason for minifying was just to strip out all the comments. One option that might be a nice middle ground would be:

```suggestion
    plugins: [
      simpleTS('./'),
      terser({
        mangle: false,  // preserve all names
        output: {
          beautify: true  // normalize/preserve whitespace
        }
      })
    ],
```